src/main/java/org/broadinstitute/dsm/db/dao/kit/BSPKitDao.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.dsm.db.dao.kit;
+
+import org.broadinstitute.ddp.db.SimpleResult;
+import org.broadinstitute.dsm.db.dao.Dao;
+import org.broadinstitute.dsm.db.dto.kit.ClinicalKitDto;
+import org.broadinstitute.dsm.statics.DBConstants;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Optional;
+
+import static org.broadinstitute.ddp.db.TransactionWrapper.inTransaction;
+
+public class BSPKitDao implements Dao<ClinicalKitDto> {
+
+    private static final String SQL_UPDATE_DUMMY_KIT = "UPDATE ddp_kit SET kit_label = ? where dsm_kit_request_id = ?";
+    private static final String SQL_SELECT_RANDOM_PT = "SELECT ddp_participant_id FROM ddp_kit_request where ddp_instance_id = ?  ORDER BY RAND() LIMIT 1";
+    private static final Logger logger = LoggerFactory.getLogger(BSPKitDao.class);
+
+    public void updateKitLabel(String kitLabel, String dsmKitRequestId) {
+        SimpleResult results = inTransaction((conn) -> {
+            SimpleResult dbVals = new SimpleResult();
+            try (PreparedStatement stmt = conn.prepareStatement(SQL_UPDATE_DUMMY_KIT)) {
+                stmt.setString(1, kitLabel);
+                stmt.setString(2, dsmKitRequestId);
+                int result = stmt.executeUpdate();
+                if (result == 1) {
+                    logger.info("Updated dummy kit, set KitLabel " + kitLabel + " for kit with dsmKitRequestId " + dsmKitRequestId);
+                }
+                else {
+                    throw new RuntimeException("Error updating kit  label for " + dsmKitRequestId + " updated " + result + " rows");
+                }
+            }
+            catch (SQLException e) {
+                dbVals.resultException = e;
+            }
+            return dbVals;
+        });
+        if (results.resultException != null) {
+            throw new RuntimeException("Error updating kit  label for " + dsmKitRequestId, results.resultException);
+        }
+    }
+
+    public String getRandomParticipantIdForStudy(String ddpInstanceId) {
```

Comment:
return Optional instead of string and null reference.
`public Optional<String> getRandomParticipantIdForStudy(String ddpInstanceId)`